exporter/tencentcloudlogserviceexporter/example/config.yaml

```diff
@@ -1,19 +1,14 @@
 receivers:
-  prometheus:
-    config:
-      scrape_configs:
-        - job_name: 'otel-collector'
-          scrape_interval: 10s
-          static_configs:
-            - targets: ['localhost:8890']
-            - targets: ['localhost:8889']
-            - targets: ['localhost:8888']
+  otlp:
+    protocols:
+      grpc:
+        endpoint: "localhost:4317"
```

Comment:
I believe the recommend approach is to use `:4317` so it will accept traffic via `0.0.0.0`